src/handlers/comment/handlers/assign.ts

```diff
@@ -55,6 +59,7 @@ export const assign = async (body: string) => {
 
   if (timeLabelsAssigned.length == 0) {
     logger.info(`No labels to calculate timeline`);
+    await addCommentToIssue("Skipping /assign since no issue labels are set to calculate the timeline", issue_number);
```

Comment:
It seems incorrect to me that you're posting the same message in two different logic branches twice.

Reply:
yes, it should say time labels on the second one.